build/changelog/build-changelog.php

```diff
@@ -4,8 +4,14 @@
 
 use Aws\Build\Changelog\ChangelogBuilder;
 
-$changelogBuilder = new ChangelogBuilder("", "", true);
+$params = [
+    'verbose_flag' => true
+];
+
+$changelogBuilder = new ChangelogBuilder($params);
```

Comment:
I can make that a todo for future. But for not we don't need this functionality as the configs are only & only used by unit tests.